### ISSUE TITLE
Create a page to allow us to upload content to the DB via API calls

### ISSUE THREAD

**Issue by georgefox6** [GillottandFoxInd/even-freds-happy#26]
We need a place to allow us to upload content such as images so that they are stored as a blob in the database.

Not sure if this is something that will make it into the production code or if it should be a standalone thing.

Need to talk it over with Josh first but might make a start on it.

**Comment by lolamathematician** [comment]
I'd like to take this ticket to learn how to integrate the db with the site.

**Comment by georgefox6** [comment]
Yeah no problem, for it to work we will need to write methods to call the API. I can make one for you as a bit of a template if you like?
